cfme/tests/webui/test_general_ui.py

```diff
@@ -366,6 +370,72 @@ def test_vm_right_size_recommendation_back_button(setup_provider, full_template_
     assert view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.meta(automates=[1627387])
+@pytest.mark.provider([BaseProvider], selector=ONE)
+def test_misclicking_checkbox_vms(appliance, setup_provider, provider):
+    """
+    Bugzilla:
+        1627387
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/8h
+        setup:
+            1. Add a provider.
+        testSteps:
+            1. Navigate to All VMs/Instances page.
+            2. Select the list view.
+            3. Click on the first column, it contains checkbox.
+            4. Assert that nothing happens and the page stays the same.
+    """
+    collection = appliance.provider_based_collection(provider)
+    view = navigate_to(collection, "All")
+    if view.toolbar.view_selector.selected != "List View":
+        view.toolbar.view_selector.select("List View")
+    table = Table(view, '//*[@id="miq-gtl-view"]//table')
+    row = next(table.rows())
```

Comment:
It would better if we add comment for why are we defining widget in test case. So that it will be more clear for newbie.

Reply:
I think it's alright, I don't see a need for defining the comment. I needed a temporary widget for the test case, so I defined it inside the test, it seems quite clear to me or would be to any newbie for that matter. If you still insist, I will add it :)